.github/workflows/test.yml

```diff
@@ -80,6 +80,10 @@ jobs:
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
           fetch-depth: 0
+      - name: Install ZLIB
```

Comment:
The CIs for MacOS and Linux do not require manual installation of ZLIB because it is already available?

Reply:
Yes, from the cmake log, zlib is available on MacOS and Ubuntu runner.